nopol/src/main/java/fr/inria/lille/repair/Main.java

```diff
@@ -111,6 +114,12 @@ public Object call() throws Exception {
 			e.printStackTrace();
 			main.showUsage();
 		}
+		if (result != null) {
+			System.out.println(result.getNopolStatus());
+
+			returnCode = (result.getPatches().isEmpty()) ? 1 : 0;
```

Comment:
status code 1 means: "Miscellaneous errors, such as "divide by zero" and other impermissible operations"
But it is not the case